src/autocomplete/shared/autocomplete-types.ts

```diff
@@ -138,3 +137,15 @@ export interface CursorPosition {
     line: number;
     column: number;
 }
+
+export interface StatementPosition {
```

Comment:
nit: I'd name it RulePosition, because there might be other. I see that it's basically imported from another file